docs/how_to_add_a_publisher.md

```diff
@@ -296,11 +298,11 @@ Additionally, attributes defined in the attribute guidelines are explicitly defi
 
 ### Extracting Attributes from Precomputed
 
-One way to extract useful information from articles rather placeholders, is to utilize `ld` and `meta` attributes of the `Article`.
+One way to extract useful information from articles rather than placeholders is to utilize the `ld` and `meta` attributes of the `Article`.
```

Comment:
"rather than placeholders" could be removed.

Reply:
I actually like that it refers to the previous procedure. What do you think?